### ISSUE TITLE
Review scrambled main RECAP documents with attachment numbers and attempt to clean them up.

### ISSUE THREAD

**Issue by albertisfu** [freelawproject/courtlistener#4600]
Once the cleanup is complete (if attachment numbers 0 and 1 are cleaned up), there should be around 395 RDs that still require cleaning.

The remaining documents that require cleaning can be found by:

```
RECAPDocument.objects.filter(
    document_type=RECAPDocument.PACER_DOCUMENT,
    attachment_number__isnull=False
).count()
```

We can inspect some of these documents to determine the best approach for cleaning them. From the examples we reviewed, the PDFs merged into some of these documents do not match, as most of them are minute entries.

**Comment by mlissner** [comment]
Alberto to review queries, then hand off to Chaco to run them.

**Comment by albertisfu** [comment]
In https://github.com/freelawproject/courtlistener/issues/4599 we confirmed the clean up is complete for these documents. This can be closed now.
